src/main/resources/static/swagger.yaml

```diff
@@ -194,4 +231,56 @@ components:
         consentArtefacts:
           type: array
           items:
-            $ref: '#/components/schemas/ConsentArtefactReference'
\ No newline at end of file
+            $ref: '#/components/schemas/ConsentArtefactReference'
+    HealthInformation:
+      type: object
+      properties:
+        limit:
+          type: integer
+        size:
+          type: integer
+        offset:
+          type: integer
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/HealthInformationRepresentation'
+    HealthInformationRepresentation:
+      type: object
+      properties:
+        1234:
```

Comment:
what is this 1234, and 7896? keys? what do these keys represent?

Reply:
These are HIP Ids.

Reply:
can we somehow describe them so? instead of marking as properties like that. maybe the key property name is just "hipId"